db/migrate/20220404042316_create_places.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePlaces < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
https://guides.rubyonrails.org/active_record_migrations.html

Reply:
yes but why is places significant to us? are they stations or landmarks? where do they fit into our database structure? should i be adding them in somewhere else

Reply:
where do places show up on the map? what are they?

Reply:
Places is not for the map - it is a migration tracker. The map currently has not been integrated in my most recent commit because I am trying to handle the controllers before I integrate, but we can assign someone the task today